FinSearchUnified/Tests/Subscriber/FrontendTest.php

```diff
@@ -96,13 +123,29 @@ public function testFrontendPreDispatchConditions($sSearch, $sCategory, $sContro
         $args->method('getRequest')->willReturn($request);
 
         // Create Mock object for Shopware Session
-        $session = $this->getMockBuilder('\Enlight_Components_Session_Namespace')
-            ->setMethods(null)
-            ->getMock();
+        $session = $this->createMock(Enlight_Components_Session_Namespace::class);
+        $session->expects($this->atLeastOnce())->method('offsetGet')->willReturnMap([
+            ['isSearchPage', $isSearch],
+            ['isCategoryPage', $isCategory],
+            ['findologicDI', false]
+        ]);
 
-        // Assign mocked session variable to application container
         Shopware()->Container()->set('session', $session);
 
+        $configArray = [
+            ['ActivateFindologic', true],
+            ['ActivateFindologicForCategoryPages', false],
+            ['findologicDI', false],
```

Comment:
`findologicDI`, `isSearchPage` and `isCategoryPage` only exist as session variables.